grammars/gfm.cson

```diff
@@ -1002,7 +1002,28 @@
         'include': 'source.clojure'
       }
     ]
-  },
+  }
+  {
+    'begin': '^\\s*(`{3,}|~{3,})\\s*(?=\\s*[-\\w]+\\s*$)'
+    'beginCaptures':
+      '0':
+        'name': 'support.gfm'
+    'end': '^\\s*\\1((?<=`)`+|(?<=~)~+)?\\s*$'
+    'endCaptures':
+      '0':
+        'name': 'support.gfm'
+    'name': 'markup.code.other.gfm'
+    'patterns': [
+      {
+        'begin': '\\G\\s*([-\\w]+).*$'
+        'beginCaptures':
+          '0':
+            'name': 'support.gfm'
+        'end': '^(?=\\s*[`~]{3,}\\s*$)'
```

Comment:
Actually, I don't think this is gonna work either, what with the new pattern-matching logic. Requiring a closing border of a minimum length kinda changes everything... because the matched length has to be checked for by the embedded patterns too.

It might be better to simply fix the `contentName` bug in first-mate instead, and do away with this hackery.